hooks/honeybadger/honeybadger.go

```diff
@@ -0,0 +1,44 @@
+package logrus_honeybadger
+
+import (
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/agonzalezro/goneybadger"
+)
+
+// HoneybadgerHook to send exceptions to an exception-tracking service compatible
+// with the Honeybadger API.
+type HoneybadgerHook struct {
+	apiKey, env string
+	timeout     time.Duration
+}
+
+func NewHoneyBadgerHook(apiKey, env string, timeout time.Duration) *HoneybadgerHook {
+	return &HoneybadgerHook{
+		apiKey:  apiKey,
+		env:     env,
+		timeout: timeout,
+	}
+}
+
+func (h *HoneybadgerHook) Fire(entry *logrus.Entry) error {
+	gb := goneybadger.NewWithTimeout(h.apiKey, h.env, h.timeout)
+	err := gb.Notify(entry.Message)
```

Comment:
Can you make the API consistent with [the Airbrake hook](https://github.com/Sirupsen/logrus/blob/master/hooks/airbrake/airbrake.go)? I.e. use `error` field if it's there, otherwise fall back to the message.